test/zmumu/tp_from_aod_MC.py

```diff
@@ -475,7 +477,7 @@
 
 if True: # turn on for tracking efficiency using L1 seeds
     process.probeL1 = cms.EDFilter("CandViewSelector",
-        src = cms.InputTag("l1extraParticles"),
+        src = cms.InputTag("gmtStage2Digis:Muon:HLT2"),
```

Comment:
Is the explicit `:HLT2` needed? 
That will not work on tranche3 samples nor on the new Moriond ones, for which the trigger is called `HLT`, and IIRC the reHLT step was meant to be configured to drop the `HLT` part and leave only the `HLT2` one, so that accessing without a process name should work on both samples.